Cargo.toml

```diff
@@ -21,6 +21,7 @@ version = "0"
 [features]
 rt = ["stm32l4x6/rt"]
 STM32L476VG = []
+STM32L496AG = []
```

Comment:
Please add to .travis.toml build that uses this feature.

Also add it for `package.metadata.docs.rs`